examples/simple-singleton/README_pt.adoc

```diff
@@ -0,0 +1,365 @@
+:index-group: Session Beans
+:jbake-type: page
+:jbake-status: status=published
+= Simples Singleton
+
+Como o nome indica, um `javax.ejb.Singleton`, é um bean de sessão com a garantia de que há no máximo uma instância no aplicativo.
```

Comment:
I think that the translation of `applicación` can be `aplicação` in pt. What do you think ?